build.rs

```diff
@@ -118,6 +118,7 @@ fn build_wrapper(opencv: pkg_config::Library) {
         "text",
         "tracking",
         "viz", // includes platform-specific headers like /usr/include/x86_64-pc-linux-gnu/opencv2/viz.hpp
+        "xfeatures2d", // only appears in some builds, maybe platform or opencv compile flag specific
```

Comment:
for future explorers who may want to reintegrate this, my opencv was compiled with:

opencv_contrib 3.4

opencv 3.4.6 using flags:
```sh
cmake \
    -DCMAKE_BUILD_TYPE=Release \
    -DBUILD_PERF_TESTS=OFF \
    -DBUILD_TESTS=OFF \
    -DINSTALL_TESTS=OFF \
    -DBUILD_DOCS=OFF \
    -DBUILD_EXAMPLES=OFF \
    -DBUILD_opencv_apps= \
    -DWITH_IPP=OFF \
    -DPYTHON_EXECUTABLE=OFF \
    -DINSTALL_PYTHON_EXAMPLES=OFF \
    -DWITH_LAPACK=ON \
    -DWITH_EIGEN=ON \
    -DBUILD_SHARED_LIBS=ON \
    -DWITH_TBB=ON \
    -DOPENCV_ENABLE_NONFREE=ON \
    -DCMAKE_INSTALL_PREFIX=/usr \
    -DOPENCV_EXTRA_MODULES_PATH=~/tools/opencv_contrib/modules \
    ..
```

on arch linux